librz/core/cmd_descs/cmd_interpret.yaml

```diff
@@ -53,4 +53,18 @@ commands:
         flags: RZ_CMD_ARG_FLAG_ARRAY
         optional: true
       - name: )
-        type: RZ_CMD_ARG_TYPE_FAKE
\ No newline at end of file
+        type: RZ_CMD_ARG_TYPE_FAKE
+  - name: ..(
+    cname: interpret_macro_multiple
+    summary: Interpret output of macro multiple times with different args
```

Comment:
maybe you want to add a `description` and some `details` as well?

Reply:
okay 436e77ca5b2ca82269ab8af66acde0442bbdc505

Reply:
okay 6423bedab4a2439eabdae0d7b5213e3cfe6cd843